### PR TITLE
feature/wordpress-rest-api

### PR DESCRIPTION
**Pull request by maeertin**
# Todo's

- [ ] **Write tests for `ApiClient` based upon the existing one in `@oakwood/accelerator`.**
*Why?* you might ask. We want to have an equal interface as the old one, in regards for easy migration for all the old projects.

**Though**, the `@oakwood` version of the `ApiClient` may obviously have flaws and that has to be taken into account when completing this version.

- [ ] **Make sure they all tests pass**

### CODE REVIEW

File: packages/api-client/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@noaignite/api-client",
```
Comment (maeertin) [inline packages/api-client/package.json:2]: Is this supposed to be `@noaignite/wordpress-rest-api`? :)

File: packages/api-client/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "api",
+    "client"
+  ],
+  "license": "MIT",
+  "main": "./src/index.ts",
+  "module": "./src/index.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/noaignite/accelerator.git",
+    "directory": "packages/wordpress-rest-api"
```
Comment (maeertin) [inline packages/api-client/package.json:18]: This should probably be `@noaignite/api-client` :)

**Comment by alexanderflink** [comment]
@maeertin can you make this a draft? This one isn't ready to be merged yet I think.

**Comment by maeertin** [comment]
Perhaps we could get assistance from @adamsoderstrom here? :)

**Comment by adamsoderstrom** [comment]
> Perhaps we could get assistance from @adamsoderstrom here? :)

I would gladly help out, @maeertin!
Are there any specified goals for this PR? :)

**Comment by alexanderflink** [comment]
Closed because Wordpress is no longer relevant for our new projects. We should look into refactoring the previous wordpress projects separately.

**Pull request closed without merging**